iam/src/utils/easPassportSchema.ts

```diff
@@ -0,0 +1,214 @@
+import {
+  NO_EXPIRATION,
+  SchemaEncoder,
+  ZERO_BYTES32,
+  MultiAttestationRequest,
+  AttestationRequestData,
+} from "@ethereum-attestation-service/eas-sdk";
+import { VerifiableCredential } from "@gitcoin/passport-types";
+import { BigNumber } from "@ethersproject/bignumber";
+
+import { fetchPassportScore } from "./scorerService";
+import { encodeEasScore } from "./easStampSchema";
+import axios from "axios";
+
+export type AttestationStampInfo = {
+  hash: string;
+  stampInfo: PassportAttestationStamp;
+  issuanceDate: BigNumber;
+  expirationDate: BigNumber;
+};
+
+export type PassportAttestationData = {
+  providers: BigNumber[];
+  info: AttestationStampInfo[];
+};
+
+export type PassportAttestationStamp = {
+  name: string;
+  index: number;
+  bit: number;
+};
+
+export type StampMetadata = {
+  id: string;
+  name: string;
+  groups: {
+    name: string;
+    stamps: {
+      name: string;
+    }[];
+  }[];
+}[];
+
+type Stamp = {
+  bit: number;
+  index: number;
+  name: string;
+};
+
+const stampMetadataEndpoint = process.env.PASSPORT_STAMP_METADATA_PATH || "";
+
+export const buildProviderBitMap = async (): Promise<Map<string, PassportAttestationStamp>> => {
+  const stampMetadata: {
+    data: StampMetadata;
+  } = await axios.get(stampMetadataEndpoint);
```

Comment:
Don't use stampMetadata from app. Create a script that generates bitmap format so that it can be versioned